### PR TITLE
Add modules for phpkafka

### PR DESCRIPTION
**Pull request by alexispeter**
Modules for https://github.com/arnaud-lb/php-rdkafka and required https://github.com/edenhill/librdkafka

### CODE REVIEW

**Comment by alexispeter** [comment]
Please review @kris-lab

**Comment by kris-lab** [comment]
- php5/Puppetfile: add `librdkafka`
- modules/librdkafka/manifests/init.pp - WARNING: double quoted string containing no variables on line 7
- `librdkafka`: is there reason we do not use DEB for this library, version constraint? https://packages.debian.org/jessie/librdkafka-dev
- modules/librdkafka/spec/init/spec.rb: use `file()` to check if file exists http://serverspec.org/resource_types.html#file

**Comment by alexispeter** [comment]
I just used 0.9.1 because the php-binding has the same version. I can try if it works with 0.8.5 if this is an issue.

**Comment by kris-lab** [comment]
Using provided DEBs from official repository is just simpler for maintenance and the puppet implementation is also more readable. If not working then lets stick to the manual installation (like is now).

**Comment by alexispeter** [comment]
please review @kris-lab

File: modules/kafka/manifests/init.pp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class kafka($version = '0.9.1') {
+
+  require 'apt'
+  require 'php5'
+
+  package { 'librdkafka-dev':
+    ensure   => present,
+    provider => 'apt',
+  }->
```
Comment (kris-lab) [inline modules/kafka/manifests/init.pp]: Just a small note: usually we put this arrow to the new line. It is more readable.

**Comment by kris-lab** [comment]
Added small note in the code.

The rest looks very good!

File: modules/kafka/manifests/init.pp [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class kafka($version = '0.9.1') {
+
+  require 'apt'
+  require 'php5'
```
Comment (kris-lab) [inline modules/kafka/manifests/init.pp]: `require 'build'` is missing?

**Comment by alexispeter** [comment]
please review and merge @kris-lab

**Comment by kris-lab** [comment]
retest this please

**Pull request merged**